kokoro/linux/dockerfile/test/csharp/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:stretch
```

Comment:
I have mixed feelings about upgrading to bullseye since that's currently "testing". The current stable is "buster". So updating to buster is fair, but I'd rather avoid updating to "bullseye".

Reply:
Updated it to use buster and added workaround. I tested that the docker file builds.

If this doesn't work then could you please finish off getting it working? I can't test whether it works end-to-end on your CI so I'm making changes and guessing if it will fix the problem or not.